src/IO/InputStreamFromROOTFile.cxx

```diff
@@ -56,6 +66,11 @@ InputStreamFromROOTFile::set_defaults()
     low_energy_window = 0.f;
     up_energy_window = 1000.f;
     read_optional_root_fields=false;
+    crystal_repeater_x = -1;
+    crystal_repeater_y = -1;
+    crystal_repeater_z = -1;
```

Comment:
Does this really need to be -1? STIR can only handle cases right now where z= 1.

Reply:
I've only moved this code from the original location in the derived classes, so no change there. I didn't check if there's actually any error checking or handling of defaults on those values. I agree it seems more logical to let them default to 1 but it doesn't seem high priority